packages/amplication-server/src/core/appSettings/dto/AdminUISettings.ts

```diff
@@ -0,0 +1,14 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
```

Comment:
I understand that the original spec included different groups for the server and the admin - but I am not sure if that is the correct structure for these parameters. 
maybe we should stick with the new structure we are using in the UI?
what do you think?

Reply:
Unrelated to Yuval's comment and probably a wider change. In NestJS' best practices, DTO file names follow this format:
```
admin-ui-settings.dto.ts
```

And the actual DTO class will be called:
```
AdminUISettingsDto
```

I think it's important that we stick to best practices, especially if we generate code for the community to follow.

You can read more here:
https://docs.nestjs.com/techniques/validation